### DISCUSSION TITLE
Question (Not an issue) | Generate patients for hospitals in multiple cities under same the organization.

### DISCUSSION THREAD

**Discussion by npagare** [synthetichealth/synthea#1427] [Q&A]
### What happened?

Hello @jawalonoski, @eedrummer and other experts,
I want to generate patients for hospitals in multiple cities under same the organization in one synthea run.

What configurations should be applied in the synthea.properties file and/or other config file(s)?

Your help will be appreciated.

Thank you,
Nilesh

### Environment

```markdown
- OS:
- Java:
```

**Comment by jawalonoski** [comment]
You shouldn't have to make any changes to do that, that is default behavior. Just `./run_synthea` for whatever state you want.

**Reply by npagare** [reply]
How to specify multiple cities each located in its own state / province for a given organization?

The need is to generate the data one execution to create a consistent dataset(s).

**Reply by jawalonoski** [reply]
When you generate a population, Synthea will use organizations near the specified area. All the patients will visit those organizations.

You don't specify a specific organization.

If you need just one, you'll have to filter or post-process the data afterwards.

**Reply by npagare** [reply]
That means create data separately for each city. Correct?

If so, how do I provide same organization as a owner or governing body for all cities located in different states / provinces?

**Reply by jawalonoski** [reply]
There is no overarching corporate structure where an organization runs multiple facilities.

Obviously this exists in the real world, but Synthea doesn't care about that or model it.

Synthea has hospitals and various facilities. If you want to link them together through some corporate organizational umbrella, that is left to you.

**Reply by npagare** [reply]
Understood. How about creating the same patient dataset for separate cities in the same state. Is this case is it possible for us to have the same organization?

Can we make that happen by using organization data from the first run and provide it as an input to the subsequent runs for cities be it in the same state or different?

I would appreciate if this would be possible in Synthea. Thanks

**Reply by jawalonoski** [reply]
If you just run one state at a time, patients in various cities will visit organizations across the state.

At this time, patients do not travel across state boundaries.

**Reply by npagare** [reply]
Jason, I agree with you from w.r.t patient's point of view.
My need is to set up an organization having hospitals across states and prepare dev-test patient base including supporting practitioners. 
I am able to generate it for a state, city. But trying to aggregate in the scenario I explained.

I understand the current design and capability. 
Should you think of a workaround, then, please share with me. I would appreciate it.

Thank you.
